frontend/__tests__/components/pod.spec.tsx

```diff
@@ -48,3 +48,59 @@ describe(PodsDetailsPage.displayName, () => {
     expect(wrapper.find(DetailsPage).exists()).toBe(true);
   });
 });
+
+describe('Renders ContainerRow', () => {
+  let pod;
+  let container;
+  let wrapper: ShallowWrapper;
+  beforeEach(() => {
+    pod = {
+      status: {
+        conditions: [],
+        containerStatuses: [],
+      },
+    };
+    container = {
+      name: 'hello-openshift',
+      image: 'aosqe/hello-openshift',
+    };
+  });
+
+  it('renders container Name in row', () => {
```

Comment:
style nit: don't capitalize "Name" etc (here and below). "in row" is redundant since we're testing `ContainerRow`.

```suggestion
  it('renders the container link', () => {
```